lib/experimental/fonts/font-library/font-library.php

```diff
@@ -79,17 +78,18 @@ function gutenberg_init_font_library_routes() {
 				'delete_others_posts'    => 'edit_theme_options',
 				'delete_published_posts' => 'edit_theme_options',
 			),
-			'map_meta_cap' => false,
-			'query_var'    => false,
+			'map_meta_cap'                   => false,
+			'query_var'                      => false,
+			'show_in_rest'                   => true,
+			'rest_base'                      => 'font-families/(?P<font_family_id>[\d]+)/font-faces',
+			'rest_controller_class'          => 'WP_REST_Font_Faces_Controller',
+			'autosave_rest_controller_class' => 'WP_REST_Autosave_Fonts_Controller',
```

Comment:
These are the new post registration settings here, using the built in plumbing for registering the routes for `wp_font_face`.